experimental/adaptive-dialog/declarative/declarative-schema.md

```diff
@@ -0,0 +1,618 @@
+
+## Rules
+_Rules_ enable you to catch and respond to events. The broadest rule is the EventRule that allows you to catch and attach a set of steps to execute when a specific event is emitted by any sub-system. Adaptive dialogs supports couple of other specialized rules to wrap common events that your bot would handle. There are a number of built in rules and you can easily create new rules.
```

Comment:
>you [](start = 98, length = 3)

nit, but I like word-wrapping these so you can easily read them outside of applications.